brayns/engine/Engine.cpp

```diff
@@ -121,4 +130,24 @@ void Engine::addCameraType(const std::string& name,
     _parametersManager.getRenderingParameters().addCamera(name);
     getCamera().setProperties(name, properties);
 }
+
+bool Engine::mustRender()
+{
+    if(_parametersManager.getApplicationParameters().getUseQuantaRenderControl())
+    {
+        // Seems to work even for animations. If not, add check:
```

Comment:
You're right, it seems strange that the `mustRender()` function return true when a simulation is playing but the camera didn't move.
Maybe the player reset the `frameBuffer->numAccumFrames` to 0. Could you figure it out?

Reply:
commiting simulation data marks the scene/model as modified, which triggers the engine to clear the accumulation frames and thus bypassing the quanta control